lib/reports.py

```diff
@@ -40,7 +40,8 @@
 GITHUB_BASE = "https://github.com"
 UPVOTE_REACTION = "\U0001f44d"
 DOWNVOTE_REACTION = "\U0001f44e"
-GITHUB_THRESHOLD = 5
+GITHUB_THRESHOLD = 5  # how many upvotes a feature req needs to be added to GitHub
```

Comment:
You might want to make these an environment variable.  Or maybe even just a default value with an overridable environment variable.